meta-refkit/recipes-devtools/qemu/qemu_%.bbappend

```diff
@@ -1,7 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/qemu:"
 
 # Replace CUSE backend with socket backend.
-SRC_URI_remove_pn-qemu-native = " \
+SRC_URI_remove = " \
```

Comment:
Yes, that's better. The pn-qemu-native was a left over from the previous experimental approach of removing it in refkit.conf.